src/sentry/dynamic_sampling/rules/combinators/base.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC, abstractmethod
```

Comment:
why do we have `combinators` module and separately `combine.py`?
Does it make sense to move `combine.py` into `combinators` folder?
Or even to `__init__.py`

Reply:
My idea was to keep `combine` at a high level because it didn't require much knowledge of the code structure to be used and it has as dependencies the combinators but nothing that extends them.